### PR TITLE
feat: application graph

### PR DESCRIPTION
**Pull request by sjaanus**
Basic graph, that works with mock data.
Next steps, work on backend and try to put more content and **styling to nodes**.
image

### CODE REVIEW

File: frontend/src/component/application/ApplicationOverview.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            },
+        ],
+    };
+
+    const applicationNode = `
+    ${applicationName}[<span>${applicationName}</span>]
+    `;
+
+    // @ts-ignore
+    window.navigateToInstances = (environment: string) => {
```
Comment (sjaanus) [inline frontend/src/component/application/ApplicationOverview.tsx:54]: This function needs to be put to window, so it is accessible to mermaid.

File: frontend/src/component/application/ApplicationOverview.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import { useRequiredPathParam } from '../../hooks/useRequiredPathParam';
+import { useNavigate } from 'react-router-dom';
+
+const StyledMermaid = styled(Mermaid)(({ theme }) => ({
+    '#mermaid .node rect': {
+        fill: theme.palette.secondary.light,
+        stroke: theme.palette.secondary.border,
+    },
+    '#mermaid .application-container': {
+        // display: 'flex',
```
Comment (sjaanus) [inline frontend/src/component/application/ApplicationOverview.tsx:15]: CSS testing, will clean up in future.

File: frontend/src/component/application/ApplicationOverview.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    },
+}));
+
+export const ApplicationOverview = () => {
+    usePageTitle('Applications - Overview');
+    const applicationName = useRequiredPathParam('name');
+    const { themeMode } = useThemeMode();
+    const navigate = useNavigate();
+
+    const app = {
```
Comment (sjaanus) [inline frontend/src/component/application/ApplicationOverview.tsx:30]: Mock data

File: frontend/src/component/application/ApplicationEdit/ApplicationEdit.tsx [hunk 4]
```diff
@@ -77,7 +76,7 @@ export const ApplicationEdit = () => {
     const tabData = [
         {
             label: 'Application overview',
-            component: <ApplicationView />,
+            component: <ApplicationOverview />,
```
Comment (thomasheartman) [inline frontend/src/component/application/ApplicationEdit/ApplicationEdit.tsx:79]: You probably only want this in the new component, right (Application.tsx). This is the old component and it's not hidden behind any flags. Of course, if it's for dev purposes, that's fine 😄

**Review by thomasheartman — APPROVED** [review]
Good starting point. Let's see how it works with more data and styling.

**Pull request merged**